cloud/shared/bin/setup.py

```diff
@@ -23,6 +24,32 @@ def run(config: ConfigLoader, params: List[str]):
     # Load Setup Class for the specific template directory
     ###############################################################################
 
+    if os.getenv('SKIP_TAG_CHECK'):
```

Comment:
This is used in our ci tests eg https://github.com/civiform/cloud-deploy-infra/blob/f72ea47cb5f0f1e67e72086991d1f9b6dc31a7e0/.github/workflows/aws-oidc-deploy-test.yaml#L19